package.json

```diff
@@ -33,7 +33,8 @@
     "snapshot": "pnpm prerelease && changeset version --snapshot preview && pnpm install && changeset publish --tag preview",
     "test": "pnpm turbo run test --scope='@nhost/*' --no-deps --include-dependencies",
     "changeset": "changeset",
-    "wait": "wait-on http://localhost:1337/v1/auth/healthz -i 500 -t 120000"
+    "wait": "wait-on http://localhost:1337/v1/auth/healthz -i 500 -t 120000",
+    "docgen": "turbo run build --scope=\"@nhost/docgen\" --no-deps && turbo run docgen --scope='@nhost/*' && :"
```

Comment:
The trailing `&& :` is necessary, because otherwise `lint-staged` would try to pass the changed files as arguments to `pnpm docgen` and stop the execution after build.

See more here: https://github.com/okonet/lint-staged/issues/174